manifests/crd.yaml

```diff
@@ -17,6 +17,24 @@ spec:
 
 ---
 
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: arangodeploymentreplications.replication.database.arangodb.com
+spec:
+  group: replication.database.arangodb.com
+  names:
+    kind: ArangoDeploymentReplication
+    listKind: ArangoDeploymentReplicationList
+    plural: arangodeploymentreplications
+    shortNames:
+    - arangorepl
+    singular: arangodeploymentreplication
+  scope: Namespaced
+  version: v1alpha
```

Comment:
other things are v1beta1, this is v1alpha.  does it matter?

Reply:
It's a choice. At some point we'll increase it.